.npmrc

```diff
@@ -1 +1,2 @@
-save-exact = false
\ No newline at end of file
+save-exact = false
+scripts-prepend-node-path = true
```

Comment:
this is the fix for this warning:

![image](https://user-images.githubusercontent.com/1477453/77657045-6556bd80-6f7d-11ea-9998-b901bd235d45.png)